test/integration/tarantool_role_test.lua

```diff
@@ -0,0 +1,172 @@
+    fio.copytree(fio.abspath('./.rocks'), fio.pathjoin(cg.router.workdir, '.rocks'))
+    cg.router:start{wait_until_ready = true}
+    fio.rmtree(tmpdir)
+end)
+
+g.after_each(function(cg)
+    cg.router:stop()
+    fio.rmtree(tmpdir)
+end)
+
+g.test_simple_role_config = function()
+    local conn = net_box.connect('localhost:3313')
+
+    conn:eval([[
+        box.schema.create_space('users', {
+            format = {
+                {name = 'id', type = 'unsigned'},
+                {name = 'first name', type = 'string'},
+                {name = 'second name', type = 'string', is_nullable = true},
+                {name = 'age', type = 'number', is_nullable = false},
+            }
+        })
+
+        box.space.users:create_index('primary', {
+            parts = {
+                {field = 1, type = 'unsigned'},
+            },
+        })
+
+        box.space.users:insert{1, 'Samantha', 'Carter', 30}
+        box.space.users:insert{2, 'Fay', 'Rivers', 41}
+        box.space.users:insert{3, 'Zachariah', nil, 13}
+        box.space.users:insert{4, 'Milo', 'Walters', 74}
+
+        box.schema.func.create('forever_true_test', {
+            body = "function(...) return true end",
+            if_not_exists = true
+        })
+    ]])
+    fiber.sleep(1)
+
+    local result = conn:eval([[
+        return box.space.users:select()
+    ]])
+    t.assert_equals(result, {})
+    conn:close()
+end
+
+g.test_waiting_for_functions = function()
+    local conn = net_box.connect('localhost:3313')
+
+    conn:eval([[
+        box.schema.create_space('users', {
+            format = {
+                {name = 'id', type = 'unsigned'},
+                {name = 'first name', type = 'string'},
+                {name = 'second name', type = 'string', is_nullable = true},
+                {name = 'age', type = 'number', is_nullable = false},
+            }
+        })
+
+        box.space.users:create_index('primary', {
+            parts = {
+                {field = 1, type = 'unsigned'},
+            },
+        })
+
+        box.space.users:insert{1, 'Samantha', 'Carter', 30}
+        box.space.users:insert{2, 'Fay', 'Rivers', 41}
+        box.space.users:insert{3, 'Zachariah', 'Peters', 13}
+        box.space.users:insert{4, 'Milo', 'Walters', 74}
+    ]])
+
+    -- function for is_expired config entry was not created, so expirationd
+    -- is waiting for it's creation.
+    fiber.sleep(1)
+    local result = conn:eval([[
+        return box.space.users:select()
+    ]])
+    t.assert_equals(result, {
+        {1, 'Samantha', 'Carter', 30},
+        {2, 'Fay', 'Rivers', 41},
+        {3, 'Zachariah', 'Peters', 13},
+        {4, 'Milo', 'Walters', 74},
+    }
+    )
+
+    -- create is_expired function and check that expirationd task started.
+    conn:eval([[
+        box.schema.func.create('forever_true_test', {
+            body = "function(...) return true end",
+            if_not_exists = true
+        })
+    ]])
+
+    fiber.sleep(2)
```

Comment:
Please, use helpers.retrying instead of just sleep:

https://github.com/tarantool/expirationd/blob/8c669678dde5f1c88e862dacb33a8e79f4ddd9a3/test/integration/role_test.lua#L97-L100

It will make the test less flaky.